includes/wikia/services/ArticleService.class.php

```diff
@@ -10,36 +10,11 @@
 class ArticleService extends WikiaObject {
 	const MAX_LENGTH = 500;
 	const CACHE_VERSION = 9;
-	const SOLR_SNIPPETS_FIELD = 'snippet_s';
```

Comment:
Can we remove this field from Solr documents?

Reply:
Sure, definitely. I'd wait with that some time after release to ensure everything is working smoothly with new snippets generation.